lib/nailed/config.rb

```diff
@@ -4,57 +4,63 @@
 
 module Nailed
   class Config
-    def self.parse_config(path_to_config = nil)
-      path_to_config ||= File.join(__dir__, "..", "..", "config", "config.yml")
-      begin
-        @@content = YAML.load_file(path_to_config) || nil
-      rescue Exception => e
-        STDERR.puts("Can't load '#{path_to_config}': #{e}")
-        exit 1
-      end
+    class << self
+      def parse_config
+        is_valid?
 
-      self.is_valid?
+        # init class variables:
+        @@organizations = []
+        (load_content['organization'] || []).each do |org|
```

Comment:
It needs to be 'organizations' (plural), this breaks the github fetch. I will fix it asap, but please let me review it before merging next time.

Reply:
See fix: https://github.com/openSUSE/nailed/pull/66